app/controllers/albums.js

```diff
@@ -0,0 +1,15 @@
+const albumsService = require('../services/albums');
+
+exports.getAlbums = (req, res, next) =>
+  albumsService
+    .albums()
+    .then(albumsData => res.send({ data: albumsData }))
+    .catch(error => next(error));
+
+exports.getAlbumPhotos = (req, res, next) =>
+  albumsService
+    .photos()
+    .then(photos =>
+      res.send.json({ data: photos.filter(photo => photo.albumId === parseInt(req.params.id)) })
```

Comment:
As we talk by Slack, `res.send` should do the trick

Reply:
Instead of doing a manual filter here, you can use queryParam like this `http://jsonplaceholder.typicode.com/photos?albumId=1`